EventKit/eventkit-cloud
# Configurable osmconvert

# Description of Improvement

Allows the memory use for osm convert to be adjusted.

## How to test

Create an OSM job, try to adjust the `OSM_MAX_TMPFILE_SIZE` to a smaller or larger amount (performance should change).

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
